### PR TITLE
Update `util` extern to v10.x

### PR DESCRIPTION
**Pull request by terurou**
Please see: https://nodejs.org/docs/latest-v10.x/api/util.html

### CODE REVIEW

File: src/js/node/util/Promisify.hx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ */
+package js.node.util;
+
+import haxe.extern.Rest;
+import haxe.Constraints.Function;
+
+@:native("util.promisify")
```
Comment (Gama11) [inline src/js/node/util/Promisify.hx]: This should probably be `@:jsRequire` like the other externs?

File: src/js/node/util/Promisify.hx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+		If `original` is not a function, `promisify()` will throw an error. If original is a function but its last
+		argument is not an error-first callback, it will still be passed an error-first callback as its last argument.
+	**/
+	@:selfCall
+	static function promisify(original:Function):Rest<Dynamic>->js.Promise<Dynamic>;
+
+	/**
+		A `<symbol>` that can be used to declare custom promisified variants of functions.
+	**/
+	#if haxe4
```
Comment (Gama11) [inline src/js/node/util/Promisify.hx]: Not sure there's much of a reason to conditionalize this, `(default,never)` also works in Haxe 3.

File: src/js/node/util/Promisify.hx (hunk 1 above)
Comment (terurou) [reply src/js/node/util/Promisify.hx]: Thanks, you are right. I fixed it.

File: src/js/node/util/Promisify.hx (hunk 2 above)
Comment (terurou) [reply src/js/node/util/Promisify.hx]: There is not `Symbol` in Haxe3.4. So I used `haxe4` flag.
Should I use `(default, never)` instead of `final`?

Comment (Gama11) [reply src/js/node/util/Promisify.hx]: Oh, I totally missed that the Haxe 3 branch uses `Dynamic`. No, this seems fine then. :)

**Comment by Gama11** [comment]
Thanks!

**Pull request merged**